Report Exporter/Sdl.Community.ReportExporter/Helpers/FolderSelectDialog.cs

```diff
@@ -11,54 +7,45 @@ public class FolderSelectDialog : IDisposable
 	{
 		private OpenFileDialog ofd;
 
+		public FolderSelectDialog()
+		{
+			ofd = new OpenFileDialog();
+			ofd.Filter = "Folders|\n";
+			ofd.AddExtension = false;
+			ofd.CheckFileExists = false;
+			ofd.DereferenceLinks = true;
+			ofd.Multiselect = false;
+		}
+
 		public string FileName
```

Comment:
Optional, poti folosi initializare explicita cum e si in [Coding Guidelines-urile](https://confluence.sdl.com/display/TPALL/Coding+Guidelines#CodingGuidelines-5.Refactoring) de la Studio:

new OpenFileDialog
{
  Filter = "Folders\\n",
  AddExtension = false,
  ...
}

Reply:
Done :)